feature/detail/src/main/java/jp/co/yumemi/droidtraining/feature/detail/components/DetailWeatherItem.kt

```diff
@@ -130,12 +146,21 @@ private fun TempItem(
     }
 }
 
+private fun Instant.formatDate(): String {
+    return LocalDateTime
+        .ofInstant(this.toJavaInstant(), ZoneId.of("UTC"))
```

Comment:
表示のタイムゾーンは特に指定ないですが、日本の方が分かりやすいです
```suggestion
        .ofInstant(this.toJavaInstant(), ZoneId.of("Asia/Tokyo"))
```